tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -14777,9 +14777,9 @@ TEST_F(EnergyPlusFixture, Test_UnitarySystemModel_SubcoolReheatCoil)
     EXPECT_EQ(state->dataCoilCooingDX->coilCoolingDXs[0].performance.OperatingMode, 1);
     EXPECT_EQ(state->dataCoilCooingDX->coilCoolingDXs[0].performance.ModeRatio, 0.0);
     EXPECT_NEAR(thisSys->LoadSHR, 0.98533, 0.001);
-    EXPECT_NEAR(thisSys->CoilSHR, 0.97702, 0.001);
+    EXPECT_NEAR(thisSys->CoilSHR, 0.98246, 0.001);
     EXPECT_NEAR(SenOutput, -2000.0, 0.5);
-    EXPECT_NEAR(LatOutput, -346.1, 0.1);
+    EXPECT_NEAR(LatOutput, -1077.6, 0.1);
```

Comment:
Now here with mode=1 I see the change I would expect except that is a huge increase in latent for such a small change in SHR.